components/Flipper/FlipperFilter/FlipperFilter.tsx

```diff
@@ -179,170 +133,262 @@ function FlipperFilter(props: Props) {
-
-                    {!props.isPremium && !freePremiumFilters ? <Tooltip type="hover" content={soldFilter} tooltipContent={nonPremiumTooltip} /> : soldFilter}
-                    {openRestrictionListDialog}
-                </div>
-
-                <div style={{ visibility: 'hidden', height: 0 }}>
-                    <Countdown key={uuids[2]} onComplete={onFreePremiumComplete} date={FREE_PREMIUM_FILTER_TIME} />
-                    <Countdown key={uuids[3]} onComplete={onFreeLoginComplete} date={FREE_LOGIN_FILTER_TIME} />
+            <div className={styles.flipperFilterGroup}>
+                <Form.Group className={styles.filterTextfield}>
+                    <Tooltip
+                        type="hover"
+                        content={
+                            <Form.Label htmlFor="min-profit" className={`${styles.flipperFilterFormfieldLabel}`}>
+                                Min. Profit:
+                            </Form.Label>
+                        }
+                        tooltipContent={
+                            <span>
+                                How much estimated profit do you at least want from each flip. Note that there is naturally more competition on higher profit
+                                flips{' '}
+                            </span>
+                        }
+                    />
+                    <NumberFormat
+                        id="min-profit"
+                        onValueChange={value => {
+                            onSettingsChange('minProfit', value.floatValue || 0)
+                        }}
+                        className={`${styles.flipperFilterFormfield} ${styles.flipperFilterFormfieldText}`}
+                        type="text"
+                        disabled={!props.isLoggedIn}
+                        isAllowed={value => {
+                            return numberFieldMaxValue(value.floatValue, 2147483647)
+                        }}
+                        customInput={Form.Control}
+                        defaultValue={flipperFilter.minProfit}
+                        thousandSeparator={getThousandSeperator()}
+                        decimalSeparator={getDecimalSeperator()}
+                        allowNegative={false}
+                        decimalScale={0}
+                    />
+                </Form.Group>
+                <div
+                    onClick={() => {
+                        setShowRestrictionList(true)
+                    }}
+                    className={styles.filterCheckbox}
+                    style={{ cursor: 'pointer' }}
+                >
+                    <span className={styles.filterBorder}>
+                        <Tooltip
+                            type="hover"
+                            content={<span className={`${styles.flipperFilterFormfieldLabel} ${styles.checkboxLabel}`}>Blacklist</span>}
```

Comment:
```suggestion
                            content={<span className={`${styles.flipperFilterFormfieldLabel} ${styles.checkboxLabel}`}>Filter-Rules</span>}
```
whitelists are often overlooked